index.html

```diff
@@ -89,7 +100,15 @@ <h2>preferCurrentTab</h2>
               >
               should now be executed, with one change - if the algorithm's execution reaches the
               step where the user agent asks the user to choose a display surface to capture, then
-              the user agent SHOULD present the current tab as the most prominent option.
+              the user agent SHOULD either:
+              <ol>
+                <li>Present the current tab as the most prominent option.</li>
+                <li>
+                  Present the current tab as the only option. In this case, the user agent MUST
+                  mitigate against clickjacking, for example by introducing a delay until user
+                  interaction may allow the capture to start.
+                </li>
+              </ol>
```

Comment:
I'd prefer weakening the recommendation here and being more explicit about its implications, perhaps along the lines of:

> ... the user agent SHOULD take the developer's preference for the current tab into account, making it the most prominent of any option presented. User agents MAY rely on this preference to limit the set of options presented to the user.
>
> Note: The carveout above is an intentional violation of [[SCREEN-CAPTURE](https://www.w3.org/TR/screen-capture/)]'s requirement that "the [getDisplayMedia](https://www.w3.org/TR/screen-capture/#dom-mediadevices-getdisplaymedia) API does not permit the use of constraints to narrow the set of options presented". This carveout does not obviate user agent's duty to ensure [active user consent](https://www.w3.org/TR/screen-capture/#dfn-active-user-consent), for example by mitigating against inadvertent sharing (clickjacking, etc).

Reply:
Adopted with minor changes. (For example, I changed "developer's preference" to "applications's preference".)